components/autocomplete/index.js

```diff
@@ -147,6 +148,17 @@ function isTextNode( node ) {
 	return node !== null && node.nodeType === 3;
 }
 
+/**
+ * Is the node an element.
+ *
+ * @param {?Node} node The node to check.
+ *
+ * @return {boolean} True if the node is an element.
+ */
+function isElement( node ) {
+	return node !== null && node.nodeType === window.Node.ELEMENT_NODE;
```

Comment:
Other files have this usually as a browser dependency. Same for above for text.

Reply:
Do you mean simply checking `nodeType`? That sounds right. I went with the pattern I saw in creating `isTextNode`, but it's unnecessary.